src/ui/components/ide/monaco-editor.tsx

```diff
@@ -100,7 +117,7 @@ export default function (props: { onChange: OnChange }) {
 		});
 
 		return dispose;
-	}, [monacoRef]);
+	}, [monacoRef, currentTheme]);
```

Comment:
It'd be best if we can create a new effect for `currentTheme` since this effect does a lot of internal stuff that we don't want to execute again when there's a theme change.